### PR TITLE
My Portfolio: Desktop Version

### PR DESCRIPTION
**Pull request by DanielAD10**
PR Description

Project Name: My Portfolio
Key Feature: Desktop Version
Tech: Built with HTML & CSS

In this PR,

Modify styling to be responsive on desktop
Add some changes in index.html
Update background images and rearrange the alignment of contact form

Thank you for reviewing my code

### CODE REVIEW

File: style.css
Comment (IngDavidTM) [inline style.css:1]: - [ ] Your work on the desktop version of your portfolio is impressive :+1: :100:. However, ensuring that the design adheres to the same background images as the Figma template is essential. Failure to do so can result in a disjointed design that could negatively impact the project's usability and aesthetic appeal. Therefore, please review the background images in your portfolio and add any necessary ones to ensure consistency with the Figma template, as required by the project :+1:.

**Your design**

Screenshot from 2023-05-11 18-39-59

Screenshot from 2023-05-11 18-42-07

Screenshot from 2023-05-11 18-43-06

**Figma**

Screenshot from 2023-05-11 18-41-00

Screenshot from 2023-05-11 18-41-48

Screenshot from 2023-05-11 18-42-55

File: style.css [hunk 1]
```diff
@@ -310,8 +323,91 @@ body {
...
+    flex-direction: column;
+    align-items: flex-start;
+    width: 650px;
+    height: 330px;
+  }
+
+  .contact-text {
+    font-size: 30px;
+    line-height: 50px;
+  }
```
Comment (IngDavidTM) [inline style.css]: - [ ] To ensure consistency with the Figma template, please align the text "I'm always..." to the middle of the form. This is important to maintain the overall design and ensure that the project meets the desired aesthetic standards :+1:. 

**Your design**

Screenshot from 2023-05-11 18-43-06

**Figma**

Screenshot from 2023-05-11 18-42-55

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @DanielAD10 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @DanielAD10, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**